kratos/solving_strategies/builder_and_solvers/residualbased_elimination_builder_and_solver.h

```diff
@@ -280,14 +276,14 @@ class ResidualBasedEliminationBuilderAndSolver
                     }
                 }
 
-		double stop_build = OpenMPUtils::GetCurrentTime();
-		if (this->GetEchoLevel() > 1 && r_model_part.GetCommunicator().MyPID() == 0)
-			std::cout << "build time: " << stop_build - start_build << std::endl;
+      double stop_build = OpenMPUtils::GetCurrentTime();
+      if (this->GetEchoLevel() > 1 && r_model_part.GetCommunicator().MyPID() == 0)
+         std::cout << "build time: " << stop_build - start_build << std::endl;

```

Comment:
good place for using info

Reply:
@loumalouomega I recommend doing what I did in #1962 regarding the timer stuff

Reply:
In these lines I just changed the "\t" to "    " but OK